src/rng.cpp

```diff
@@ -5,35 +5,49 @@
 #include <algorithm>
 #include <utility>
 
-long rng( long val1, long val2 )
+#include "cata_utility.h"
+
+static std::uniform_int_distribution<int> rng_int_dist;
+static std::uniform_real_distribution<double> rng_real_dist;
+static std::normal_distribution<double> rng_normal_dist;
+static std::uniform_int_distribution<unsigned int>
```

Comment:
I'm a little uncomfortable about having these be global statics; seems like it risks static initialization order bugs -- calling an RNG inside the initialization of another global is a plausible thing to happen.  Function-local statics would be safe and only a small performance hit.

Reply:
It turns out they're each called from one method each, so it's trivial.